README.md

```diff
@@ -271,3 +273,20 @@ Sent event format: `Event category` / `Event action` / `Event label`
 
 7. User updates their reason of reply request
   - `Article` / `ProvidingReason` / `<articleId>`
+  - When the LIFF opens, page view for page `/reason` is also sent.
+
+8. User opens article list
+  - Page view for page `/articles` is sent
+  - If opened via rich menu: `utm_source=rumors-line-bot&utm_medium=richmenu`
+  - If opened via push message: `utm_source=rumors-line-bot&utm_medium=push`
+  - If opened via LINE notify: `utm_source=line-notify&utm_medium=push`
```

Comment:
According to [discussion](https://g0v.hackmd.io/OC7BneJ3TEGJHge1hWV-0w#Cofacts-Next-%E8%BF%BD%E8%B9%A4), we don't open articles page via LINE Notify anymore.

Reply:
Nice catch! Thanks.